ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.ts

```diff
@@ -196,12 +210,37 @@ export const findDragonMarks = (
 const triggerSet: TriggerSet<Data> = {
   id: 'TheUnendingCoilOfBahamutUltimate',
   zoneId: ZoneId.TheUnendingCoilOfBahamutUltimate,
+  config: [
+    {
+      id: 'heavensfallTowerPosition',
+      comment: {
+        en:
+          `With a tower at Nael being position 1, rotating clockwise, your tower poisition. e.g. H1 in <a href="https://clees.me/guides/ucob/" target="_blank">Clees guide</a> is position 7.`,
```

Comment:
```suggestion
          `With a tower at Nael being position 1, rotating clockwise, your tower position. e.g. H1 in <a href="https://clees.me/guides/ucob/" target="_blank">Clees guide</a> is position 7.`,
```
Might also want to add an apostrophe, as "Clees' guide", or reword it as "this guide by Clees".